.test-defs/bastion-test.yaml

```
@@ -0,0 +1,22 @@
+kind: TestDefinition
+metadata:
+  name: bastion-test
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Integration test for bastion creation and deletion
+  activeDeadlineSeconds: 1800
+
+  command: [bash, -c]
+  args:
+  - >-
+    go test -timeout=0 -mod=vendor ./test/integration/bastion
+    --v -ginkgo.v -ginkgo.progress -ginkgo.noColor
```

```suggestion
    --v -ginkgo.v -ginkgo.progress -ginkgo.no-color
```

Ref https://github.com/gardener/gardener-extension-provider-openstack/pull/410

---

committed per comments 43a97228